### DISCUSSION TITLE
Can't renew education pack through student verification

### DISCUSSION THREAD

**Discussion by efeichen** [community/community#102461] [GitHub Education]
### Hi everyone,

I'm having an issue where I receive this message after I submitted my school email through the verification page to try to renew my student pack. The last verified date is empty, and I have not been approved for several weeks since last time I checked.

<img width="747" alt="image" src="https://github.com/community/community/assets/16854657/37a77458-248a-42d4-a02e-8343b1996024">

There was a similar issue with another user mentioned here: https://github.com/orgs/community/discussions/83533. Hope I can get some help as well. Thank you!

**Comment by naveen-98** [comment]
Hello @efeichen ! 🙂

Here are some general steps you can take to address this issue:

1. **Contact GitHub Support:**
   Reach out to GitHub Support for assistance. You can submit a request through the [GitHub Support Contact Form](https://support.github.com/contact) and provide them with details about the issue you are facing. They will be able to look into your account-specific information.

2. **Check Email Verification:**
   Ensure that the email address associated with your GitHub account, especially your school email, is verified. Sometimes, verification emails may be sent to your inbox or spam folder, and you need to confirm the email address.

3. **Review GitHub Documentation:**
   Check the [GitHub Student Developer Pack documentation](https://education.github.com/pack) for any specific guidance or updates related to the verification process. GitHub may provide information on common issues and how to resolve them.

4. **Follow Up on Previous Discussions:**
   Since you mentioned a similar issue being discussed in the GitHub Community discussions, monitor that thread for updates or solutions. GitHub staff or other community members may provide guidance.

Remember to provide as much detail as possible when reaching out to GitHub Support, including the steps you've taken, any error messages, and relevant account information. They will be better equipped to assist you with account-specific issues.
